opencti-platform/opencti-front/src/private/components/common/form/CreatorField.tsx

```diff
@@ -74,49 +96,72 @@ const CreatorField: FunctionComponent<CreatorFieldProps> = ({
         const NewCreators = (
           (data as CreatorFieldSearchQuery$data)?.members?.edges ?? []
         ).map((n) => ({
-          label: n?.node.name,
+          label: n?.node.name ?? t_i18n('Unknown'),
           value: n?.node.id,
+          extra: getExtraFromNode(n?.node),
         }));
-        const templateValues = [...creators, ...NewCreators];
+        const templateValues = [...creatorOptions, ...NewCreators];
         // Keep only the unique list of options
         const uniqTemplates = templateValues.filter((item, index) => {
           return (
             templateValues.findIndex((e) => e.value === item.value) === index
           );
         });
-        setCreators(uniqTemplates);
+        setCreatorOptions(uniqTemplates);
       });
   };
-
   return (
     <div style={{ width: '100%' }}>
       <Field
         component={AutocompleteField}
         name={name}
         textfieldprops={{
           variant: 'standard',
-          label: t_i18n(label),
-          helperText: helpertext,
+          label,
           onFocus: searchCreators,
         }}
+        disableClearable
         onChange={onChange}
         style={containerStyle}
         noOptionsText={t_i18n('No available options')}
-        options={creators}
-        isOptionEqualToValue={isOptionEqualToValue}
+        options={creatorOptions}
+        isOptionEqualToValue={(option: CreatorOption, selected: CreatorOption) => option.value === selected.value}
         onInputChange={searchCreators}
         renderOption={(
           props: React.HTMLAttributes<HTMLLIElement>,
-          option: { color: string; label: string },
+          option: CreatorOption,
         ) => (
-          <li {...props}>
-            <div className={classes.icon} style={{ color: option.color }}>
-              <ItemIcon type="user" />
-            </div>
-            <div className={classes.text}>{option.label}</div>
+          <li {...props} >
+            <Box
```

Comment:
You could use `div` instead of MUI component `Box`. In our case Box just render a div but with performance cost. From the MUI documentation:
![image](https://github.com/OpenCTI-Platform/opencti/assets/33682259/5ae45f08-3d8b-41f2-a213-15537d5b56ce)
https://mui.com/system/getting-started/usage/#performance-tradeoffs

Reply:
ok sure. When shall we use Box instead then ? only if we need theme-related styles ?

Reply:
I have no idea, never been in a situation where I need this component...